doc/contributing/docs/markup/code.rst

```diff
@@ -1,8 +1,49 @@
 Writing about code
 ==================
 
+When writing articles, you have to format code specially, separating it from
+other text. This document will guide you through typical cases when
+it is recommended to use code highlighting.
+
+Defining what code is
```

Comment:
The general rule is that code is any text, processed by a machine. Any parts of a programming language — yes. Package names and console commands — yes. Error messages in the GUI which humans have to read — probably, no. Error messages in strictly formatted application logs — probably, yes. Product names — certainly no.